Models/CompanyProperties.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace DIS_projekt.Models
+{
+    public class CompanyProperties
```

Comment:
Model se naziva po onome što opisuješ. Ili Company ili Employee ili nešto. Ne znam što misliš pod CompanyProperties.